man/docker.1.md

```diff
@@ -19,42 +19,42 @@ To see the man page for a command run **man docker <command>**.
 
 # OPTIONS
 **--help**
-  Print usage statement
+  Print usage statement.
 
-**--config**=""
-  Specifies the location of the Docker client configuration files. The default is '~/.docker'.
+**--config** ""
+  Specifies the location of the Docker client configuration files. The default is `~/.docker`.
 
-**-D**, **--debug**=*true*|*false*
-  Enable debug mode. Default is false.
+**-D**, **--debug** *true*|*false*
+  Enable debug mode. Default is *false*.
```

Comment:
```suggestion
**-D**, **--debug** **true**|**false**
￼  Enable debug mode. Default is **false**.
```

As true and false are literals, they should be rendered in bold, not italic. Italic is for variable things and file names.